woocommerce/woocommerce
# E2e/e2e merchant settings add shipping classes

### All Submissions:






### Changes proposed in this Pull Request:


This Pull Request adds a new e2e test for the "Merchant / Settings / Add Shipping Classes" critical flow.
Closes #29005.

### How to test the changes in this Pull Request:

1. Run the `runAddShippingClassesTest` test both in headful and headless mode and see if it passes.

### Other information:
